server/chat-commands/info.ts

```diff
@@ -1991,36 +1991,39 @@ export const commands: ChatCommands = {
 		target = target.toLowerCase().trim();
 		const showAll = target === 'all';
 		if (showAll && this.broadcasting) {
-			return this.sendReplyBox("You cannot broadcast all FAQs at once.");
+			return this.sendReplyBox(this.tr`You cannot broadcast all FAQs at once.`);
 		}
 
+		const autoconfirmedFAQ = `A user is autoconfirmed when they have won at least one rated battle and have been registered for one week or longer. In order to prevent spamming and trolling, most chatrooms only allow autoconfirmed users to chat. If you are not autoconfirmed, you can politely PM a staff member (staff have %, @, or # in front of their username) in the room you would like to chat and ask them to disable modchat. However, staff are not obligated to disable modchat.`;
```

Comment:
Is there a reason this was pulled out? I think it makes more sense where it is.

Reply:
It was pulled out because of eslint's line length rules -- #7175 will make it possible to put it back.

Reply:
I mean, it can still be in the line above where it's used, instead of way up here. I think it's generally more readable that way.

For this specifically, yes, I'll merge #7175 so you don't need to worry about this. But for future reference, this sort of constant should just be declared the line above where it's used, for best readability.

Reply:
Alright, I'll keep that in mid.